src/pages/PostsList/components/PostPreview.jsx

```diff
@@ -0,0 +1,42 @@
+import { LikeOutlined } from '@ant-design/icons'
+import { Button, List, Space, Typography } from 'antd'
+import React from 'react'
+
+const { Paragraph } = Typography
+
+const IconText = ({ icon, text }) => (
+  <Space>
+    {React.createElement(icon)}
+    {text}
+  </Space>
+)
```

Comment:
Please move it to separate file

Reply:
To powinien być osobny komponent?

Reply:
Yes, overall we avoid to declare more than one component inside one file.